code/game/machinery/computer/robot_control.dm

```diff
@@ -11,6 +11,7 @@
 	light_color = LIGHT_COLOR_PURPLE
 
 	var/safety = 1
+	var/detonate_cooldown
```

Comment:
```suggestion
	var/static/detonate_cooldown
```
Make it static to persist across all consoles - otherwise you *will* get powergames who screwdriver and unscrewdriver the console to reset the cooldown.

Reply:
oh lovely.